### PR TITLE
WiP: Spot sdk

### PR DESCRIPTION
**Pull request by zbettenbuk**

### CODE REVIEW

File: react/features/spot/components/SpotControllerView.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    /**
+     * Implements {@code PureComponent#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const { _device, _headerStyles, _visible } = this.props;
+
+        return (
+            <Modal
```
Comment (saghul) [inline react/features/spot/components/SpotControllerView.js]: Why not a dialog?

File: react/features/spot/components/SpotDevicesListView.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    /**
+     * Implements {@code PureComponent#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        return (
+            <Modal
```
Comment (saghul) [inline react/features/spot/components/SpotDevicesListView.js]: ditto

File: ios/Podfile.lock [hunk 3]
```diff
@@ -42,7 +42,7 @@ PODS:
     - FirebaseAnalytics (~> 5.1)
     - FirebaseAnalyticsInterop (~> 1.0)
     - FirebaseCore (~> 5.2)
-  - FirebaseInstanceID (3.7.0):
+  - FirebaseInstanceID (3.8.1):
```
Comment (saghul) [inline ios/Podfile.lock]: Why are all these updated? Did you run `pod update` ? Instead, please do `pod install --repo-update` which shouldn't update unrelated dependencies.

File: react/features/spot/components/SpotControllerView.js (hunk 1 above)
Comment (zbettenbuk) [reply react/features/spot/components/SpotControllerView.js]: It's the Jitsi SlidingView, not actually a Modal. Check the import

Comment (saghul) [reply react/features/spot/components/SpotControllerView.js]: Ah, gotcha, disregard then!

**Pull request closed without merging**